instrumentation/opentelemetry-instrumentation-system-metrics/setup.cfg

```
@@ -42,6 +42,7 @@ packages=find_namespace:
 install_requires =
     opentelemetry-api ~= 1.11
     opentelemetry-sdk ~= 1.11
+    psutil >= 5.9
```

We don't know everything works without breaking in upcoming major 6.x 
```suggestion
    psutil ~= 5.9
```